include/hip/nvcc_detail/hip_runtime_api.h

```diff
@@ -772,6 +772,13 @@ inline static hipError_t hipGetDeviceProperties(hipDeviceProp_t* p_prop, int dev
     p_prop->maxSharedMemoryPerMultiProcessor = cdprop.sharedMemPerMultiprocessor;
     p_prop->isMultiGpuBoard = cdprop.isMultiGpuBoard;
 
+    p_prop->maxTexture1D    = cdprop.maxTexture1D;
```

Comment:
To maintain backward compatibility, I think you just need to condition these writes with HIP patch version.

Reply:
> To maintain backward compatibility, I think you just need to condition these writes with HIP patch version.

sorry this does not work.